apps/site/src/lib/components/ValidatingForm/ValidatingForm.tsx

```diff
@@ -25,10 +29,15 @@ function ValidatingForm(props: PropsWithChildren<FormProps>) {
 		<form
 			onSubmit={handleSubmit}
 			noValidate // use custom validation feedback
-			className={validated ? styles.validated : styles.notYetValidated}
+			className={`bg-white p-5 md:p-10 rounded-2xl mx-5 text-black ${
```

Comment:
Suggestion: styling should be the responsibility of the parent, not the form itself, which is why we had the styles on the consumers.